modules/sdk-coin-atom/test/unit/atom.ts

```diff
@@ -397,16 +398,23 @@ describe('ATOM', function () {
     const destinationAddress = wrwUser.destinationAddress;
     const coin = coins.get('tatom');
     const testBalance = '150000';
-    const testAccountNumber = '123';
-    const testSequenceNumber = '0';
-    const testChainId = 'test-chain';
+    const testAccountNumber = '725163';
+    const testSequenceNumber = '8';
+
+    const testBalance1 = '150000';
```

Comment:
we can use the same value in both the cases as it is only used while creating stub